src/unix/bsd/apple/mod.rs

```diff
@@ -2070,6 +2070,11 @@ pub const LC_SEGMENT_64: u32 = 0x19;
 pub const MH_MAGIC: u32 = 0xfeedface;
 pub const MH_MAGIC_64: u32 = 0xfeedfacf;
 
+// net/if_utun.h
+pub const UTUN_CONTROL_NAME: &'static str = "com.apple.net.utun_control";
```

Comment:
Unfortunately static strings don't map greatly to C right now and we don't have any other instances of `str` right now, could this be left out for now?

Reply:
okay.